packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -52,7 +52,7 @@ export class CallContext {
         cameras: [],
         microphones: [],
         speakers: [],
-        unparentedViews: new Map<LocalVideoStreamState, LocalVideoStreamState>()
```

Comment:
this used to be a map of an object.. to itself?

Reply:
(where key and value the same here?)

Reply:
never mind read the comment where this is defined

Reply:
Yeah, this was a very weird map. I had to double and triple check that we didn't actually depend on the keys being referentially equal across some API calls.